### DISCUSSION TITLE
Green dot

### DISCUSSION THREAD

**Discussion by Furim** [plausible/analytics#1206] [Self-Hosted Support] [answered]
i posted script on my website but i still see green dot

**Comment by Furim** [comment, marked as answer]
nevermind, if u have ssl on your website the stats must have ssl certifcate too


basically script works but when website have https:// and stats does not have https its not working(its just not posting the script on website)

**Reply by kalipso-cyber** [reply]
Could you specify which "stats" file you mean? I'm having the same issue (only see green dot despite adding script between HTML head tags of my index.html) and I also have SSL configured. I've included the HTTPS in the base URL of my /hosting/plausible-conf.env.

In the beginning, the page wouldn't even load the script. Double-checking, I noticed it was trying to load the script directly from my host, but since the file never got placed there to begin with, it obviously resulted in a 404. I manually changed the snippet to point to plausible.io/js/plausible.js, and now at least it loads. 

By now, the script loads fine, but I still don't get any visits.

**Reply by Furim** [reply]
as stats i mean the script that is posted on  website and is related to your website for example mydomain.xyz:8844 and if the domain which is related to stats dont have ssl and your website that u post script on have ssl certificate it just wont load, thats how my plausible-conf.env looks like check if same as yours


```
ADMIN_USER_EMAIL=idkwhatismygmail@gmail.com
ADMIN_USER_NAME=user
ADMIN_USER_PWD=password
BASE_URL=https://subdomain.domain.xyz
DISABLE_AUTH=false
DISABLE_REGISTRATION=false
SECRET_KEY_BASE=secret
```

also your domain that is plausible on have ssl as ssl certificate or u mean that u just included https

**Reply by kalipso-cyber** [reply]
Thanks for your reply. I had set up SSL on the subdomain and entered the BASE_URL with https, so I would think that it should have worked. However, I became a little frustrated with the process, so I decided to reinstall Plausible completely. Unfortunately, I am know facing some issues in that the Plausible instance isn't even accessible via the browser anymore. I always get a "Connection reset" error when trying to access it.

I think this might be related to my whole subdomain and different port setup, but I haven't been able to figure out what it is exactly just yet. Don't have so much time for the project, but I will post back once everything is solved or I have a more concrete question.

Again, thanks for helping!
